docs/in_flight_exit_scenarios.md

```diff
@@ -241,7 +241,7 @@ response:
         "competing_txbytes": "F317010180808080940000...",
         "competing_input_index": 1,
         "competing_sig": "9A23010180808080940000...",
-        "competing_txid": 2600003920012,
+        "competing_tx_pos": 2600003920012,
```

Comment:
I'd like to know how to decode it to `{blknum, txindex}`

Reply:
You should simply use `UTXO.Position`, but in this case `oindex` field is not important, though using `0` as its value is recommended.